yarn.lock

```diff
@@ -0,0 +1,276 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why does this file exist at the top level? Did you accidentally yarn install there? This should go away.